src/configuration/FirestoreConfigurationContext.tsx

```diff
@@ -49,6 +55,7 @@ type ConfigurationContextState = {
   fareProductTypeConfigs: FareProductTypeConfig[];
   travelSearchFilters: TravelSearchFiltersType | undefined;
   appTexts: AppTexts | undefined;
+  profileInfoUrls: ProfileInfoUrls | undefined;
```

Comment:
I thin I would prefer just calling this "urls" or "links" in the firestore-configuration, to be able to reuse it for all links in the app.

Reply:
Agree, renamed to `configurableLinks` and keeping `urls` in Firestore.